pkg/controller/kataconfig/openshift_reconciler.go

```diff
@@ -16,6 +16,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	nodeapi "k8s.io/api/node/v1beta1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	apimachineryErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
`k8s.io/apimachinery/pkg/api/errors` is getting imported here and the line above too.

Reply:
will fix it!